packages/elements-dev-portal/src/components/NodeContent/index.tsx

```diff
@@ -69,7 +69,11 @@ const LinkComponent: React.FC<{ node: { url: string } }> = ({ children, node: {
     // Resolve relative file URI with
     const resolvedUri = resolve(dirname(node.uri), url);
     const [resolvedUriWithoutAnchor, hash] = resolvedUri.split('#');
-    const edge = node.outbound_edges.find(edge => edge.uri === url || edge.uri === resolvedUriWithoutAnchor);
+    const decodedUrl = window.decodeURI(url);
+    const decodedResolvedUriWithoutAnchor = window.decodeURI(resolvedUriWithoutAnchor);
```

Comment:
1. You may want to do `decodeURIComponent` instead, don't you?
2. `window` will break SSR. Node does have a `decodeURIComponent` so either just getting rid of the prefix or preferrably `globalThis.decodeURIComponent` will do the job.

Reply:
Do I?

I was under the impression that `decodeURIComponent` was meant for a single elements within a path like `hello`, while `decodeURI` was for paths like `hello/world`?

Those are relative urls, but still urls nonetheless?

Reply:
I think so. Whether they are URLs or not is pretty much irrelevant in this situation IMO. You want to simply URL-decode the entire string. (Which happens to contain a relative path, but that's a coincidence.) [`decodeURI`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURI) does some magic which you probably don't want.